src/clp_ffi_py/ir/PyLogEvent.hpp

```diff
@@ -0,0 +1,144 @@
+#ifndef CLP_FFI_PY_PY_LOG_EVENT_HPP
```

Comment:
Typically automated tools will generate the string `CLP_FFI_PY_PYLOGEVENT_HPP`. Can we stick with that convention? cc: @davidlion